actionlib_tutorials/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ generate_messages(DEPENDENCIES std_msgs actionlib_msgs)
 
 ## Define catkin exports
 catkin_package(
-  CATKIN_DEPENDS roscpp actionlib
```

Comment:
I see.

Then the reason why you're dropping `roscpp` from `CATKIN_DEPENDS` is we don't need to export roscpp headers in this package (sorry I'm not catkin-cmake ex-poet)?